x-pack/test/functional/apps/rollup_job/hybrid_index.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+export default function ({ getService, getPageObjects }) {
+
+  const log = getService('log');
+  const es = getService('es');
+  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'indexManagement']);
+
+  const createRollupJobBody = {
+    'index_pattern': 'sensor-*',
+    'rollup_index': 'sensor_rollup',
+    'cron': '9,19,29,39,49,59  * * * * ?',
+    'page_size': 1000,
+    'groups': {
+      'date_histogram': {
+        'field': 'timestamp',
+        'fixed_interval': '1h',
+        'delay': '10s'
+      },
+      'terms': {
+        'fields': ['node']
+      }
+    },
+    'metrics': [
+      {
+        'field': 'temperature',
+        'metrics': ['min', 'max', 'sum']
+      },
+      {
+        'field': 'voltage',
+        'metrics': ['avg']
+      }
+    ]
+  };
+
+  console.log(log, es, createRollupJobBody);
+  describe('rollup jobs', function () {
```

Comment:
Another test already has this same `describe('rollup jobs'...)` /x-pack/test/functional/apps/rollup_job/rollup_jobs.js

That `rollup jobs` should probably be in the describe in the parent index file, and then these test files should have a unique describe text that helps you understand which file it's in.  So this one should be something like `hybrid index`